src/ResourceManagement/Network/ExpressRouteCircuitPeeringsImpl.cs

```diff
@@ -0,0 +1,133 @@
+        public IEnumerable<Microsoft.Azure.Management.Network.Fluent.IExpressRouteCircuitPeering> List()
+        {
+            return WrapList(Extensions.Synchronize(() => Inner.ListAsync(parent.ResourceGroupName, parent.Name)));
+        }
+
+        public async override Task<IPagedCollection<IExpressRouteCircuitPeering>> ListByParentAsync(string resourceGroupName, string parentName, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return await PagedCollection<IExpressRouteCircuitPeering, ExpressRouteCircuitPeeringInner>.LoadPage(
+                async (cancellation) => await Inner.ListAsync(resourceGroupName, parentName, cancellation),
+                WrapModel, cancellationToken);
+        }
+
+        ///GENMHASH:1F414E796475F1DA7286F29E3E27589D:AFCD61B362D6BFBAB04EC3872A0EDD43
+        public override async Task DeleteByParentAsync(string groupName, string parentName, string name, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            await Inner.DeleteAsync(groupName, parentName, name, cancellationToken);
+        }
+
+        ///GENMHASH:ECAE06A61F07DA1F3D1A8D023A8B2918:DDB2BB043185E9D0C3266308196FB6CC
+        public ExpressRouteCircuitPeeringImpl DefineMicrosoftPeering()
+        {
+            return new ExpressRouteCircuitPeeringImpl(parent, new ExpressRouteCircuitPeeringInner(), Inner, ExpressRouteCircuitPeeringType.MicrosoftPeering);
+        }
+
+        ///GENMHASH:21D478F33B98E8BA003B0CA8A77DC927:E43675D6B0AF2246CC7B27D4E8ADDC2C
+        public ExpressRouteCircuitPeeringImpl DefineAzurePublicPeering()
+        {
+            return new ExpressRouteCircuitPeeringImpl(parent, new ExpressRouteCircuitPeeringInner(), Inner, ExpressRouteCircuitPeeringType.AzurePublicPeering);
+        }
+
+
+        ///GENMHASH:5C58E472AE184041661005E7B2D7EE30:6B6D1D91AC2FCE3076EBD61D0DB099CF
+        public IExpressRouteCircuitPeering GetByName(string name)
+        {
+            return Extensions.Synchronize(() => GetByNameAsync(name));
+        }
+
+        ///GENMHASH:921F345925153FFB7B665161B62A21A2:EF8777BA4B3D684F2BED0D68FF019106
+        public ExpressRouteCircuitPeeringImpl DefineAzurePrivatePeering()
+        {
+            return new ExpressRouteCircuitPeeringImpl(parent, new ExpressRouteCircuitPeeringInner(), Inner, ExpressRouteCircuitPeeringType.AzurePrivatePeering);
+        }
+
+        /// <return>An observable emits packet captures in this collection.</return>
+        ///GENMHASH:7F5BEBF638B801886F5E13E6CCFF6A4E:DBC1BFDB73D33E089DF6B74E82CCBAA9
+        public async Task<Microsoft.Azure.Management.ResourceManager.Fluent.Core.IPagedCollection<IExpressRouteCircuitPeering>> ListAsync(bool loadAllPages = true, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            //$ return wrapPageAsync(Inner.ListAsync(parent.ResourceGroupName(), parent.Name()));
+
+            return null;
```

Comment:
Please check this method as you have commented java code here. Is it by design to return null here?